Assets/_BForBoss/_Weapons/Scripts/Bullets/BulletTypes.cs

```diff
@@ -5,4 +5,10 @@ public enum BulletTypes
         NoPhysics,
         Physics
     }
+
+    public enum BulletOwner
```

Comment:
so instead of having another source of Player/enemy - since we have layers and tags for players already.

we can have 2 separate bullet spawners - one for enemies and another for players. I would say it's more organized and cleaner that way.